### PR TITLE
New package: seq24-0.9.3

### PR DESCRIPTION
**Pull request by DavideMaggio**
#### Testing the changes
- I tested the changes in this PR: **YES**

#### Local build testing
- I built this PR locally for my native architecture, (x86_64-musl)
- I built this PR locally for these architectures (if supported. mark crossbuilds):
  - armv7l-musl
  - armv7l
  - x86_64
  - i686

### CODE REVIEW

File: srcpkgs/seq24/template [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+revision=1
+build_style=gnu-configure
+configure_args="--disable-lash"
+hostmakedepends="pkg-config alsa-lib-devel automake"
+makedepends="gtkmm2-devel alsa-lib-devel jack-devel"
+short_desc="Pattern based midi sequencer with strong live performance functions"
+maintainer="mag <mag-one@autistici.org>"
+license="GPL-2.0-or-later"
+homepage="http://filter24.org/seq24/"
+distfiles=https://launchpad.net/seq24/trunk/${version}/+download/${pkgname}-${version}.tar.bz2
```
Comment (biopsin) [inline srcpkgs/seq24/template:13]: Quotation mark missing on url

File: srcpkgs/seq24/template [hunk 2]
```diff
@@ -0,0 +1,18 @@
+# Template file for 'seq24'
+pkgname=seq24
+version=0.9.3
+revision=1
+build_style=gnu-configure
+configure_args="--disable-lash"
+hostmakedepends="pkg-config alsa-lib-devel automake"
+makedepends="gtkmm2-devel alsa-lib-devel jack-devel"
```
Comment (paper42) [inline srcpkgs/seq24/template:8]: We should be trying to get rid of gtk2 packages, not add more of them. Is there a plan to support gtk3/4 or move to another toolkit?

Comment (DavideMaggio) [reply srcpkgs/seq24/template:8]: uh...understand... I recently used this to resurrect some old big midi sessions, it worked very well so I shared my template. 
It's an 'old' sequencer, I think there are not many future plans but we'll see. Thanks.

Comment (paper42) [reply srcpkgs/seq24/template:8]: Even if we don't end up merging this PR, the template is still here and interested people can build it on their own, so thanks for that!

**Pull request closed without merging**